src/Controls/src/Core/HybridWebView/HybridWebView.cs

```diff
@@ -183,13 +106,7 @@ private void AsyncTaskCompleted(string taskId, string result)
 				return default;
 			}
 			return JsonSerializer.Deserialize<TReturnType?>(stringResult, returnTypeJsonTypeInfo);
```

Comment:
Maybe this can/should be `DeserializeAsync`?

```suggestion
			return await JsonSerializer.DeserializeAsync<TReturnType?>(stringResult, returnTypeJsonTypeInfo);
```